haproxy/src/configure.py

```diff
@@ -33,6 +33,10 @@
 DOWN_INTER = os.environ.get('DOWN_INTER', INTER)
 RISE = os.environ.get('RISE', '2')
 FALL = os.environ.get('FALL', '3')
+FRONTEND_SSL_ENABLED = os.environ.get('FRONTEND_SSL_ENABLED', 'false').lower() == 'true'
+FRONTEND_SSL_PORT = os.environ.get('FRONTEND_SSL_PORT', '6000')
+FRONTEND_SSL_PEM = os.environ.get('FRONTEND_SSL_PEM', '/etc/ssl/certs/cert.pem')
```

Comment:
You are right, there should be a default. But I would prefer to provide a dummy certificate instead of generating one on the fly.

Otherwise I would need to add OpenSSL to the image and the certificate will be as bad as a static dummy PEM file I copy to the image.

The intended use case is for the user to provide his own certificate by mounting a volume. I updated the documentation accordingly.